### ISSUE TITLE
NSG Trojan programs missing updated art

### ISSUE THREAD

**Issue by Rintarou** [mtgred/netrunner#7887]
somewhat related to [#7001 ] and [#7005 ]

The Trojan subtype was added to some programs, which has become very relevant with the new card set.
though the implementation of these programs was errata'd to Trojan, a handful still use the old art.
I propose to switch those few to the updated art.

Affected cards are:
- Botulus
- Egret
- Tranquilizer

    
Based on comparison between NetrunnerDB and the Jinteki card search

**Comment by NBKelly** [comment]
Yeah I don't think we've switched to the remastered art for anything except ashes so far.

I'll see about switching us over to the remastered art for gateway and system update (did they really remaster that? very cool if so) some time later this year.
